### DISCUSSION TITLE
Is there a way to autocomplete issue numbers in commit descriptions?

### DISCUSSION THREAD

**Discussion by tmcw** [jesseduffield/lazygit#3894] [Q&A]
This might be asking for a "pony", but in some previous tools I've used around git in vim - using [vim-rhubarb](https://github.com/tpope/vim-rhubarb), I could type `#` and get autocomplete for issue numbers in the repository that matches with the currently-checked out project. Is such a thing possible in lazygit, or in combination with other tools?

**Comment by mark2185** [comment]
Could you give a bit more information what you had in mind?

E.g. if it's for commit messages, you could write those in the editor and have access to `vim-rhubarb`. Or if it's something `github` specific, you could list the issues with `gh issue list` and use that through a custom command or something.
